src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -2,24 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Globalization;
-using System.IO;
 using System.Linq;
 using System.Reflection;
-using System.Runtime.CompilerServices;
 using System.Security;
 using System.Threading;
 using System.Xml;
+
 using DataContractDictionary = System.Collections.Generic.Dictionary<System.Xml.XmlQualifiedName, System.Runtime.Serialization.DataContract>;
 
 namespace System.Runtime.Serialization
 {
     internal sealed class ClassDataContract : DataContract
     {
+        internal const string ContractTypeString = "ClassDataContract";
+        public override string? ContractType => ContractTypeString;
+
```

Comment:
Would `nameof(ClassDataContract)` be a better option?

Reply:
Maybe? This was the hackiest part of rejoining the separated projects. Super proud of this one. 😬